brave-tests/src/test/java/brave/internal/recorder/PendingSpansClassLoaderTest.java

```diff
@@ -15,6 +15,7 @@
 
 import brave.handler.FinishedSpanHandler;
 import brave.handler.MutableSpan;
+import brave.handler.SpanCollector;
```

Comment:
eventhough nostalgia for `SpanCollector` `SpanHandler` may still be better name I admit as we can't move the `MutableSpan` out of this package without certain chaos...

Reply:
Also `collector` can cause confusion with server-side collectors.

Just for reference, and to add one more alternative naming, in Kafka Clients [_interceptors_](https://cwiki.apache.org/confluence/display/KAFKA/KIP-42%3A+Add+Producer+and+Consumer+Interceptors) have a similar function.

Reply:
good point on collector being conflated (I think  I recall now we dodged the word collector in the first place due to this..)

personally I like handler better, as usually interceptor means getting in the middle of something. ex

```java
void intercept(thing, chain){
  return chain.proceed(thing);
}
```

Reply:
Fair enough. I also prefer Handler instead of Listener..--naming is hard :)